### DISCUSSION TITLE
9-generics/

### DISCUSSION THREAD

**Discussion by giscus** [teivah/100-go-mistakes#41] [Discussions]
# 9-generics/

None

https://100go.co/9-generics/

**Comment by krishnateja262** [comment]
Awesome, thank you, learned about  ~

**Comment by Dzelix** [comment]
Did not completely get what ``` <2>``` means in line ```var keys []K <2>``` of  function getKeys.

**Reply by teivah** [reply]
Fixed, it was an old code annotation.

**Comment by vincenshen01** [comment]
Thanks, I learned.
